direct/io-kafka/src/main/java/cz/o2/proxima/direct/kafka/KafkaLogReader.java

```diff
@@ -432,7 +434,7 @@ private void submitConsumerWithObserver(
                   }
                 }
               }
-              increaseWatermarkOnEmptyPolls(emptyPollCount, partitionToClockDimension, clock);
```

Comment:
Looks like we can completely remote the `partitionToClockDimension` variable?

Reply:
Hmm I hope that it is removed, isn't it?